tests/utils_test.py

```diff
@@ -60,6 +64,68 @@ def test_find_endpoints():
     assert np.allclose(p, [[10, 0], [0, 10]])
 
 
+def test_get_drape_model(tmp_path):
+    ws = Workspace(os.path.join(tmp_path, "test.geoh5"))
+    x = np.arange(11)
+    y = -x + 10
+    locs = np.c_[x, y, np.zeros_like(x)]
+    h = [0.5, 0.5]
+    depth_core = 5.0
+    pads = [0, 0, 0, 0]  # [5, 5, 3, 1]
+    expansion_factor = 1.1
+    model, mesh, sorting = get_drape_model(  # pylint: disable=W0632
+        ws,
+        "drape_test",
+        locs,
+        h,
+        depth_core,
+        pads,
+        expansion_factor,
+        return_colocated_mesh=True,
+        return_sorting=True,
+    )
+    ws.close()
+    resorted_mesh_centers = mesh.cell_centers[sorting, :]
+    model_centers = compute_alongline_distance(model.centroids)
+    model_centers[:, 0] += h[0] / 2
+    assert np.allclose(model_centers, resorted_mesh_centers)
+
+
+def test_cell_centers_to_faces():
+    test_faces = [0, 3, 5, 6, 7, 8, 10, 13]
+    centers = pd.DataFrame(test_faces).rolling(2).mean().values[1:].flatten()
```

Comment:
Let's try to not use Pandas if we don't have to - pretty heavy library to import for just a rolling mean. Consider using our running mean method under `geoapps.driver_base.utils.running_mean`